dashboard/src/containers/ServiceItemContainer/ServiceItemContainer.ts

```diff
@@ -0,0 +1,50 @@
+import * as _ from "lodash";
+import { connect } from "react-redux";
+import { Action } from "redux";
+import { ThunkDispatch } from "redux-thunk";
+
+import actions from "../../actions";
+import ServiceItem from "../../components/AppView/ServicesTable/ServiceItem";
+import { Kube } from "../../shared/Kube";
+import { IResourceRef, IStoreState } from "../../shared/types";
+
+interface IServiceItemContainerProps {
+  serviceRef: IResourceRef;
+}
+
+function mapStateToProps({ kube }: IStoreState, props: IServiceItemContainerProps) {
+  const { serviceRef } = props;
+  const serviceKey = Kube.getResourceURL(
+    serviceRef.apiVersion,
+    "services",
+    serviceRef.namespace,
+    serviceRef.name,
+  );
+  return {
+    ...props,
```

Comment:
I'd be more specific on what we pass here. Just for clarity. 

I wrote it in another comment but I am not sure why you need to pass `serviceRef` to the component.

Reply:
I use `serviceRef` in `ServiceItem` to display the name of the Service whilst fetching (the screenshot doesn't show it, but the name is shown right away, with a loading icon for the rest of the rows. It's not very important, but I like being able to see what resources are loading.

Reply:
<img width="1009" alt="screen shot 2019-01-10 at 13 09 13" src="https://user-images.githubusercontent.com/1544861/50970313-fb1d3e80-14d8-11e9-8408-0b8919f4574f.png">

Reply:
Now that I think about it, I can change this to just send a `name` prop to the ServiceItem component, since it doesn't need the full `serviceRef`!